docs/source/articles/configure-test-factories.html.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Configure Test Factories
```

Comment:
Would something more like "Working with Test Factories" and explaining why they're there, what they do, etc make sense?

Reply:
In the latest commit I added "Factories" as a subsection of "Testing Overview" to provide a place to explain factories. And I added the additional howto doc "Add a Test Factory" to provide a procedure for that specific task, since @jyucis singled it out as an important test related task.